help/invite-new-users.md

```diff
@@ -29,9 +29,11 @@ permission to invite users.
 
 {start_tabs}
 
-{!invite-users.md!}
```

Comment:
I think we want these instructions to be directly for the **Invite users** option in gear menu. Going to the **Organization settings** menu is at least a couple of extra clicks.